CHANGELOG.md

```diff
@@ -17,7 +17,11 @@ and this project adheres to
   be deduced from the available emitted metrics.
 - [#4360](https://github.com/firecracker-microvm/firecracker/pull/4360): Added
   dev-preview support for backing a VM's guest memory by 2M hugetlbfs pages.
-  Please see the [documentation](docs/hugepages.md) for more information.
+  Please see the [documentation](docs/hugepages.md) for more information
+- [#4486](https://github.com/firecracker-microvm/firecracker/pull/4486): Added
+  sync block and net device metrics for backend latencies and queue backlog
+  lengths, which can be used to analyse saturation of the Firecracker VMM thread
+  and device backends.
```

Comment:
nit: for me the `backend` word is associated with `vhost-user-backed`. Maybe a bit rephrased version like this is better:
`Added metrics for processing latency and queue length in (sync)block and net devices. These metrics can be used to analyze saturation of the Firecracker VMM thread when these devices are in use.`

Reply:
rephrased to avoid work backend.